example/src/classes/view/view.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Базовый класс для всех элементов, которые будут являться элементами пользовательского интерфейса,
+ * т.е. которые должны что-либо отображать на экране.
+ */
+export default class View {
```

Comment:
This class is useless at the moment. In our base component class we should encapsulate all logic that works with html. Create all needed methods like `append` for appending instances of such class, `setTextContent` to set content of element.
We should pass an object to the constructor of this class, which would contain:
```
{
    tagName = 'div',
    classNames = [],
    textContent = '',
    parentNode,
  } = DEFAULT_PROPERTIES
```